ci-operator/config/redhat-chaos/krkn-hub/redhat-chaos-krkn-hub-main__4.14-nightly-x86-64.yaml

```diff
@@ -40,7 +40,7 @@ tests:
     pre:
     - chain: ipi-aws-pre
     - chain: create-infra-move-ingress-monitoring-registry
-    workflow: redhat-chaos-krkn-hub-tests
+    workflow: redhat-chaos-pvc-scenarios
```

Comment:
shouldn't we run both tests here? like you did below?

Reply:
Fixed it. Thanks.